### DISCUSSION TITLE
KeepassXC plugin can't connect to database.

### DISCUSSION THREAD

**Discussion by ChusVa** [keepassxreboot/keepassxc#11208] [Q&A]
OS Linux Cinnamon, browser Firefox,  KeepassXC 2.7.9. In the "Connected Databases" settings, when I click the "Connect" button, nothing happens. What's the point of the plugin if it doesn't work?

**Comment by varjolintu** [comment]
Check these steps:
https://keepassxc.org/docs/KeePassXC_UserGuide#_setup_browser_integration
https://github.com/keepassxreboot/keepassxc-browser/wiki/Troubleshooting-guide

If your browser is installed via Flatpak, it's not supported.
